webpack/stories/docs/debounce-middleware.stories.mdx

```diff
@@ -0,0 +1,20 @@
+import { Meta } from '@theforeman/stories';
+
+<Meta
+  title="Introduction|Debounce Middleware"
```

Comment:
Should we do `title="Introduction|Redux Middlewares/Debounce"` to handle more middleware in the future?